GliaWidgets/Sources/ViewController/Chat/ChatViewController.Mock.swift

```diff
@@ -366,6 +367,53 @@ extension ChatViewController {
         return controller
     }
 
+    // MARK: - Glia Virtual Assistant Persistent Button State
+
+    static func mockGvaPersistentButton() throws -> ChatViewController {
+        var chatViewModelEnv = ChatViewModel.Environment.mock
+        chatViewModelEnv.loadChatMessagesFromHistory = { true }
+
+        let messageUuid = UUID.incrementing
+        let messageId = { messageUuid().uuidString }
+        let queueId = UUID.mock.uuidString
+
+        let jsonData = mockGvaPersistentButtonJson()
+        let metadataWrapper = try JSONDecoder().decode(MetadataWrapper.self, from: jsonData ?? Data())
+        let metadata = Message.Metadata(container: metadataWrapper.container)
+
+        let messages: [ChatMessage] = [
+            .mock(id: messageId(),
+                  queueID: queueId,
+                  operator: .mock(
+                    name: "Rasmus",
```

Comment:
Here the indentation seems to be only with two spaces instead of four

Reply:
Fixed [6f13d33](https://github.com/salemove/ios-sdk-widgets/pull/705/commits/6f13d33b94f07d0cef99fe6ac5657c3da740a4e0)